### PR TITLE
Fix group capabilities

### PR DESCRIPTION
**Pull request by javfg**
This PR updates the frontend to accommodate the changes in https://github.com/cs3org/reva/pull/4400. It also updates the winspaces position and name in the sidebar.

### CODE REVIEW

File: packages/web-app-files/src/index.ts [hunk 1]
```diff
@@ -115,6 +104,18 @@ const navItems = [
     enabled(capabilities) {
       return true
     }
+  },
```
Comment (diocas) [inline packages/web-app-files/src/index.ts:107]: Why did you decide to separate this menu again? Didn't we agree to have it next to EOS projects?

Maybe @glpatcern knows?

**Review by diocas — COMMENTED** [review]
This (the user group capabilities) should be pushed to oC but for now, I'm fine with everything as is. But I have a question about the new placement of the menu.

File: packages/web-app-files/src/index.ts (hunk 1 above)
Comment (javfg) [reply packages/web-app-files/src/index.ts:107]: Exactly! It was suggested by @glpatcern. It makes sense, as what you get inside the winspaces explorer is a list of letters just like in the eos explorer.

Comment (diocas) [reply packages/web-app-files/src/index.ts:107]: Understood. Just to document here :)

**Review by diocas — APPROVED** [review]

**Comment by diocas** [comment]
@elizavetaRa can you tag and deploy a new version of web?

**Pull request merged**